packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -67,10 +67,11 @@ export interface ClientEvent {
 }
 
 export interface BehavioralEvent {
-  // TODO: not implemented
-  name: 'host.meeting.participant.admitted' | 'sdk.media-flow.started';
-  payload?: never;
-  options?: never;
+  context: any;
+  name: string;
+  payload: any;
```

Comment:
payload also probably should have a type although i understand why payload isn't typed currently